index.bs

```diff
@@ -301,9 +301,9 @@ XRFrame {#xrframe-interface}
 <pre class="idl">
 partial interface XRFrame {
     XRJointPose? getJointPose(XRJointSpace joint, XRSpace baseSpace);
-    void fillJointRadii(sequence&lt;XRJointSpace&gt; jointSpaces, Float32Array radii);
```

Comment:
unsure if this one needs to be a bool too

Reply:
Might as well, for consistency. It's not going to be as frequently used as the return value from fillPoses, but it'll definitely bite someone if the return value is different between them.